src/main/java/net/glowstone/entity/GlowPlayer.java

```diff
@@ -1,9 +1,5 @@
 package net.glowstone.entity;
 
-import static com.google.common.base.Preconditions.checkArgument;
```

Comment:
Revert the changes done to imports in this file.

Reply:
Can I rewrite the history for these commits that I've created?

Reply:
I rewrote the history in my fork, because I thought it would be okay, hopefully it is.